src/celengine/category.h

```diff
@@ -21,17 +21,18 @@ class UserCategory
 
 private:
     std::string m_name;
+    const char *m_i18n;
```

Comment:
std::string m_i18nname;

we won't have 100500 categories, so no need to optimize memory usage *here*